### ISSUE TITLE
A question about "post_ops" in mixmatch.py

### ISSUE THREAD

**Issue by chencjGene** [google-research/mixmatch#24]
Hi, I do not how what is the "post_ops" in the following codes for? could you help explain for it?

`batches = layers.interleave([x] + y, batch)`
        `skip_ops = tf.get_collection(tf.GraphKeys.UPDATE_OPS)`
        `logits = [classifier(batches[0], training=True)]`
        `post_ops = [v for v in tf.get_collection(tf.GraphKeys.UPDATE_OPS) if v not in skip_ops]`

**Comment by david-berthelot** [comment]
In this code snippet, post_ops collects the operations to update batch norm estimates of mean and variance. Tensorflow automatically puts them inside the `tf.GraphKeys.UPDATE_OPS` collection.

**Comment by chencjGene** [comment]
Got it! Thanks.
